bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -143,20 +144,13 @@ trait GemsStrategy extends AgsStrategy {
     // Create a set of the angles to try.
     val anglesToTry = (0 until 360 by 45).map(Angle.fromDegrees(_)).toSet
 
-    // A way to terminate the Mascot algorithm immediately in the following cases:
-    // 1. A usable 2 or 3-star asterism is found; or
-    // 2. If no asterisms were found.
-    // Returning false will stop the search
-    def progress(s: Strehl, usable: Boolean): Boolean = {
-      !((usable && s.stars.size >= 2) || (s.stars.size < 2))
-    }
-
-    // Iterate over 45 degree position angles if no asterism is found at PA = 0.
-    val gemsCatalogResults = results.map(result => GemsResultsAnalyzer.analyzeGoodEnough(ctx, anglesToTry, result, progress))
+    // Iterate over 45 degree position angles if no 3-star asterism is found at PA = 0.
+    val gemsCatalogResults = results.map(result => GemsResultsAnalyzer.analyzeGoodEnough(ctx, anglesToTry, result, _.stars.size < 3))
```

Comment:
This is definitely much easier to process.  It does change the meaning from what was installed before though.  If you're sure this is correct, 👍 Either way we need them to do mascot testing with the OT as well since this is used by both just to be sure.